salt/cloud/clouds/scaleway.py

```diff
@@ -78,8 +78,9 @@ def avail_images(call=None):
             '-f or --function, or with the --list-images option'
         )
 
-    items = query(method='images')
+    items = query(method='images', root='marketplace_root')
     ret = {}
+    log.debug("items type: %s", type(items))
```

Comment:
This seems like this was used while making these changes? does it still need to be here, it doesn't seem useful.

Reply:
yep, removed